cmake/modules/FindUSD.cmake

```diff
@@ -159,13 +159,12 @@ find_package_handle_standard_args(USD
     REQUIRED_VARS
         PXR_USD_LOCATION
         USD_INCLUDE_DIR
```

Comment:
I think you should also replace  `USD_INCLUDE_DIR` by `PXR_INCLUDE_DIRS`. I don't think hydra even uses USD_INCLUDE_DIR so no other changes would be required.